features/facebookImport/cypress/integration/basic_spec.js

```diff
@@ -0,0 +1,66 @@
+import "../../dist/pod";
+import commonStructure from "../../src/static/commonStructure";
+import { BlobWriter, ZipWriter, TextReader } from "@zip.js/zip.js";
+import Storage from "../../src/model/storage.js";
+
+const noDataFileName = "no-data.txt";
+let blob;
+let storage;
+let pod;
+
+before(async () => {
+    pod = await window.pod;
+    const zipBlobWriter = new BlobWriter("application/zip");
+    const zipWriter = new ZipWriter(zipBlobWriter);
+    for (let key in commonStructure) {
```

Comment:
Nit: Seems like you could use `for (let [key, value] of Object.entries(commonStructure)` here.